OpenRA.Mods.Common/Widgets/Logic/Lobby/LobbyUtils.cs

```diff
@@ -438,9 +438,9 @@ public static void SetupSpawnWidget(Widget parent, Session.Slot s, Session.Clien
 		public static void SetupEditableReadyWidget(Widget parent, Session.Slot s, Session.Client c, OrderManager orderManager, MapPreview map)
 		{
 			var status = parent.Get<CheckboxWidget>("STATUS_CHECKBOX");
-			status.IsChecked = () => orderManager.LocalClient.IsReady || c.Bot != null;
+			status.IsChecked = () => orderManager.LocalClient.IsReady;
 			status.IsVisible = () => true;
```

Comment:
To elaborate: we need more logic to implement what we discussed earlier. Just don't do the OS specific code part.

On 2015-07-12, at 10:22 PM, Deniz <notifications@github.com> wrote:

> In OpenRA.Mods.Common/Widgets/Logic/Lobby/LobbyUtils.cs:
> 
> > @@ -438,9 +438,9 @@ public static void SetupSpawnWidget(Widget parent, Session.Slot s, Session.Clien
> >  		public static void SetupEditableReadyWidget(Widget parent, Session.Slot s, Session.Client c, OrderManager orderManager, MapPreview map)
> >  		{
> >  			var status = parent.Get<CheckboxWidget>("STATUS_CHECKBOX");
> > -			status.IsChecked = () => orderManager.LocalClient.IsReady || c.Bot != null;
> > +			status.IsChecked = () => orderManager.LocalClient.IsReady;
> >  			status.IsVisible = () => true;
> STATUS_CHECKBOX is defined as Visible: false in yaml and set as true here. Is there a reason for not removing both of them?
> 
> ―
> Reply to this email directly or view it on GitHub.
>